test/integration/targets/asa_og/tests/cli/asa_og.yaml

```diff
@@ -0,0 +1,547 @@
+---
+- name: Playbook for asa_og test.
```

Comment:
@FedericoOlivieri This integration tests should be a set of tasks and not a playbook itself. Could you please fix this?

Reply:
Is now done